ui/pages/swaps/swaps.util.js

```diff
@@ -440,15 +453,14 @@ export async function fetchSwapsFeatureFlags() {
   return response;
 }
 
-export async function fetchTokenPrice(address) {
-  const query = `contract_addresses=${address}&vs_currencies=eth`;
-
+export async function fetchTokenPrice(address, currency = 'eth') {
```

Comment:
Might want someone from swaps to look over these changes to their functions

Reply:
I would assume there is another way to do this using a controller.